x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -429,6 +429,11 @@ public enum Cap {
          */
         KQL_FUNCTION(Build.current().isSnapshot()),
 
+        /**
+         * Hash function
+         */
+        HASH_FUNCTION(Build.current().isSnapshot()),
```

Comment:
Are we ok just releasing this as is without keeping it on a snapshot build first?

Reply:
I am not entirely familiar with the release procedure re esql functions.
This one looks fairly simple so maybe? But I am looking for input from others on this question.

Reply:
*generally* it's fine to just release it to the wild. If, for example, you were uncomfortable with the syntax or list of supported hashes. I scanned this PR this morning and saw you talking about the list of hashes being jvm dependent which isn't a surprise but is kind of annoying. So if you are worried about it then, yeah, we can keep this a snapshot. But if not, :shrug:.

The real thing is to ask yourself, "what's left before I remove this?" If the list doesn't fit in one PR, then, yeah. This or a feature flag is the way to go. Frankly I think a feature flag is better, but it's not a big deal either way.